enacts/flex_fcst/maproom.py

```diff
@@ -42,52 +40,6 @@ def register(FLASK, config):
 
     APP.layout = layout.app_layout()
 
-    def adm_borders(shapes):
-        with psycopg2.connect(**GLOBAL_CONFIG["db"]) as conn:
-            s = sql.Composed(
-                [
-                    sql.SQL("with g as ("),
-                    sql.SQL(shapes),
-                    sql.SQL(
-                        """
-                        )
-                        select
-                            g.label, g.key, g.the_geom
-                        from g
-                        """
-                    ),
-                ]
-            )
-            df = pd.read_sql(s, conn)
-
-        df["the_geom"] = df["the_geom"].apply(lambda x: wkb.loads(x.tobytes()))
-        df["the_geom"] = df["the_geom"].apply(
-            lambda x: x if isinstance(x, MultiPolygon) else MultiPolygon([x])
-        )
-        shapes = df["the_geom"].apply(shapely.geometry.mapping)
-        for i in df.index: #this adds the district layer as a label in the dict
-            shapes[i]['label'] = df['label'][i]
-        return {"features": shapes}
-
-
-    def make_adm_overlay(adm_name, adm_sql, adm_color, adm_lev, adm_weight, is_checked=False):
-        border_id = {"type": "borders_adm", "index": adm_lev}
-        return dlf.Overlay(
-            dlf.GeoJSON(
-                id=border_id,
-                data=adm_borders(adm_sql),
-                options={
-                    "fill": True,
-                    "color": adm_color,
-                    "weight": adm_weight,
-                    "fillOpacity": 0,
-                },
-            ),
-            name=adm_name,
-            checked=is_checked,
-        )
-
-
     #Should I move this function into the predictions.py file where I put the other funcs?
     #if we do so maybe I should redo the func to be more flexible since it is hard coded to read each file separately..
     def read_cptdataset(lead_time, start_date, y_transform=config["y_transform"]):
```

Comment:
Just noticing that the logic for reading CPT datasets is written here, not imported from cptio. Not something to chnage in this PR, but potentially something to add to the todo list.